main.py

```diff
@@ -431,6 +433,32 @@ async def list_result_directory() -> List[int]:
     return rid_list
 
 
+class ResultFileType(str, Enum):
+    h5 = "h5"
+    code = "code"
+    vcd = "vcd"
```

Comment:
Please use UPPERCASE as pylint complains.

Reply:
Oh, I'm sorry that I didn't check the Pylint complains. I will fix it.